src/volksbot_node.cpp

```diff
@@ -204,7 +204,12 @@ class ROSCall
 void ROSCall::velCallback(const geometry_msgs::Twist::ConstPtr& msg)
 {
   last_cmd_vel_time_ = ros::Time::now();
-  volksbot_.set_wheel_speed(msg->linear.x - axis_length_ * msg->angular.z * 0.5, msg->linear.x + axis_length_ * msg->angular.z * 0.5);
+  double max_vel = volksbot_.get_max_vel();
+  double velocity = msg->linear.x;
+  
+  velocity = std::min(max_vel, msg->linear.x); 
+  velocity = std::max(-max_vel, msg->linear.x); 
+  volksbot_.set_wheel_speed(velocity - axis_length_ * msg->angular.z * 0.5, msg->linear.x + axis_length_ * msg->angular.z * 0.5);
```

Comment:
Should be `volksbot_.set_wheel_speed(velocity - axis_length_ * msg->angular.z * 0.5, velocity + axis_length_ * msg->angular.z * 0.5);`